css/style.css

```
@@ -39,34 +45,173 @@ html {
 }
 
 .hero__title {
+  margin-bottom: 16px;
   font-size: 22px;
   line-height: 32px;
-  margin-bottom: 16px;
 }
 
 .hero__button-container {
-  width: 100%;
   margin-bottom: 20px;
+  width: 100%;
 }
 
 .hero__button {
-  background: linear-gradient(90deg, #fa1660, #ff6e00);
-  border-radius: 4px;
-  color: var(--white);
+  background: var(--purple);
+  color: var(--text-input-field);
   width: 100%;
   cursor: pointer;
-  height: 50px;
+  letter-spacing: 1px;
+  margin-top: 1px;
+  border-radius: 4px;
   font-size: 15px;
   line-height: 15px;
-  letter-spacing: 1px;
+  height: 50px;
   max-width: 540px;
-  margin-top: 1px;
   text-transform: uppercase;
 }
 
 .hero__link {
   color: var(--text-input-field);
-  font-size: 14px;
   letter-spacing: 0.2px;
+  line-height: 26px;
+  font-size: 14px;
+  margin: 0 0 24px;
+}
+.hint {
+  -webkit-animation: bounce 1.75s infinite;
+  animation: bounce 1.75s infinite;
+  background-color: transparent;
+  border: none;
+  position: absolute;
+  cursor: pointer;
+  font-size: 0;
+  margin: 0 auto;
+  opacity: .65;
+  bottom: 2vw;
+  padding: 6px;
+  height: 48px;
+  width: 48px;
+  fill: #f9f9f9;
+  pointer-events: none;
+  left: 50%;
+  top: 92%;
+  transform: translate(-50%, -50%);
+}
+
+.plans {
+  background-color: var(--background);
+  padding: 10vw 5vw;
+  
+}
+
+.plans__title {
+  color: var(--text-input-field);
+  letter-spacing: normal;
+  text-align: center;
+  margin-top: 0;
+  margin-bottom: 24px;
+  font-size: 26px;
+  line-height: 36px;
+}
+
+.plan__title {
+  color: var(--gray);
+  font-size: 15px;
+  line-height: 25px;
+}
+
+.plans__options {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding-left: 0;
+  margin: 0 -12px;
+}
+
+.plan {
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position: top;
+  box-sizing: border-box;
+  height: auto;
+  padding-bottom: 40px;
+  max-width: 380px;
+  width: 100%;
+  margin-top: 0;
+  padding: 60px 40px 0;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+  text-align: center;
+  justify-content: space-between;
+}
+
+.plan--combo {
+  background-image: url('../img/promo.png');
+  margin-bottom: 40px;
+}
+
+.plan--monthly {
+  background-image: url('../img/month-year.png');
+  margin-top: 40px;
+  margin-bottom: 40px;
```

I think you can reuse these properties since it is being used in different selectors
```suggestion
  margin-top: 40px;
  margin-bottom: 40px;
```